### PR TITLE
Fix quickadd error

### LINKED ISSUE #349: Console error in production
This pops up when you hover over the quick submit form on the landing page.

```
rollbar.min.js:2 Realm app hydration failed: Storage is missing a provider type
r.<computed> @ rollbar.min.js:2
e @ bundle.dom.es.js:4553
37444 @ realmApp.js:6
s @ bootstrap:19
82588 @ UserContext.js:16
s @ bootstrap:19
40973 @ realmApp.js:8
s @ bootstrap:19
18840 @ wrap-page.js:12
s @ bootstrap:19
68238 @ gatsby-browser.js:3
s @ bootstrap:19
65182 @ api-runner-browser-plugins.js:2
s @ bootstrap:19
97343 @ api-runner-browser.js:1
s @ bootstrap:19
89917 @ page-renderer.js:42
s @ bootstrap:19
(anonymous) @ wrapNativeSuper.js:36
s.O @ chunk loaded:25
t @ jsonp chunk loading:84
(anonymous) @ framework-a4a2dc3ac4fd1ec9c23e.js:2
prefetch.js:37 GET https://incidentdatabase.ai/page-data/apps/quickadd/page-data.json net::ERR_ABORTED 404
(anonymous) @ prefetch.js:37
i @ prefetch.js:17
(anonymous) @ prefetch.js:71
c @ prefetch.js:65
t.doPrefetch @ loader.js:417
n.doPrefetch @ loader.js:518
t.prefetch @ loader.js:405
enqueue @ loader.js:561
n._prefetch @ index.js:179
n @ index.js:221
(anonymous) @ index.js:108
(anonymous) @ index.js:101
prefetch.js:37 GET https://incidentdatabase.ai/page-data/apps/quickadd/page-data.json net::ERR_ABORTED 404
(anonymous) @ prefetch.js:37
i @ prefetch.js:17
(anonymous) @ prefetch.js:71
c @ prefetch.js:65
t.doPrefetch @ loader.js:417
n.doPrefetch @ loader.js:518
t.prefetch @ loader.js:405
enqueue @ loader.js:561
n._prefetch @ index.js:179
n @ index.js:221
(anonymous) @ index.js:108
(anonymous) @ index.js:101
```

### PR DESCRIPTION
**Pull request by cesarvarela**
Closes #349

### CODE REVIEW

File: site/gatsby-site/content/research.mdx [hunk 1]
```diff
@@ -31,7 +31,7 @@ Every incident has its own suggested citation that credits both the submitter(s)
 
 **Add Single Incident Reports**
 
-We currently accept incident reports in a [quick add](/apps/quickadd) form, which does not credit submitters, and the [submit app](/apps/submit) which allows you to assign credit for the submission. Both types of submissions are a great help to preventing repeating history.
+We currently accept incident reports in a [quick add](/apps/submit) form, which does not credit submitters, and the [submit app](/apps/submit) which allows you to assign credit for the submission. Both types of submissions are a great help to preventing repeating history.
```
Comment (smcgregor) [inline site/gatsby-site/content/research.mdx:34]: `We currently accept incident reports in a [quick add](/apps/submit) form` -> `We currently accept incident reports in a quick add form on the [landing page](/)`

File: site/gatsby-site/src/pages/apps/submitted.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ const SubmittedIncidentsPage = ({ data, ...props }) => {
         <h1>Quick Add URLs</h1>
         <p>
           These reports were added anonymously by users in the{' '}
-          <Link to="/apps/quickadd"> Quick Add </Link> form.
+          <Link to="/apps/submit"> Quick Add </Link> form.
```
Comment (smcgregor) [inline site/gatsby-site/src/pages/apps/submitted.js:41]: `<Link to="/apps/submit"> Quick Add </Link> form` -> `Quick Add form on the landing page`

**Comment by smcgregor** [comment]
I left a few comments to update the messaging on this.

**Comment by cesarvarela** [comment]
Updated

**Pull request merged**